example/cr-detailed.yaml

```diff
@@ -61,26 +61,25 @@ spec:
     # See https://github.com/pravega/pravega/blob/3f5b65084ae17e74c8ef8e6a40e78e61fa98737b/config/config.properties
     # for available configuration properties
     options:
-      pravegaservice.containerCount: "4"
-      pravegaservice.cacheMaxSize: "1073741824"
-      pravegaservice.zkSessionTimeoutMs: "10000"
-      attributeIndex.readBlockSize: "1048576"
-      readIndex.storageReadAlignment: "1048576"
-      durableLog.checkpointMinCommitCount: "300"
-      bookkeeper.bkAckQuorumSize: "3"
-      metrics.dynamicCacheSize: "100000"
-      metrics.enableStatistics: "true"
-      metrics.statsdHost: "telegraph.default"
-      metrics.statsdPort: "8125"
+      pravegaservice.container.count: "4"
+      pravegaservice.cache.size.max: "1073741824"
+      pravegaservice.zk.connect.sessionTimeout.milliseconds: "10000"
+      readindex.storageRead.alignment: "1048576"
+      durablelog.checkpoint.commit.count.min: "300"
+      bookkeeper.ack.quorum.size: "3"
+      metrics.dynamicCache.size: "100000"
+      metrics.statistics.enable: "true"
+      metrics.statsD.connect.host: "telegraph.default"
+      metrics.statsD.connect.port: "8125"
       # The mount dir for secrets is /etc/secret-volume
-      controller.auth.tlsEnabled: "true"
-      controller.auth.tlsCertFile: "/etc/secret-volume/controller01.pem"
-      controller.auth.tlsKeyFile: "/etc/secret-volume/controller01.key.pem"
-      controller.rest.tlsKeyStoreFile: "/etc/secret-volume/controller01.jks"
-      controller.rest.tlsKeyStorePasswordFile: "/etc/secret-volume/password"
-      pravegaservice.enableTls: "true"
-      pravegaservice.certFile: "/etc/secret-volume/segmentStore01.pem"
-      pravegaservice.keyFile: "/etc/secret-volume/segmentStore01.key.pem"
+      controller.security.tls.enable: "true"
+      controller.security.tls.server.certificate.location: "/etc/secret-volume/controller01.pem"
+      controller.security.tls.server.privateKey.location: "/etc/secret-volume/controller01.key.pem"
+      controller.security.tls.server.keyStore.location: "/etc/secret-volume/controller01.jks"
+      controller.security.tls.server.keyStore.pwd.location: "/etc/secret-volume/password"
+      pravegaservice.security.tls.enable: "true"
+      pravegaservice.security.tls.server.certificate.location: "/etc/secret-volume/segmentStore01.pem"
+      pravegaservice.security.tls.server.privateKey.location: "/etc/secret-volume/segmentStore01.key.pem"
 
```

Comment:
You might want to add `auth` related params as well in here.

Reply:
done